Backpack-SwiftUI/ImageGalleryGrid/Classes/BPKImageGalleryCategory.swift

```diff
@@ -0,0 +1,52 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import SwiftUI
+
+enum BPKImageGalleryCategoriesTypeContainer<ImageView: View> {
+    case chip(_ chipCategories: [BPKImageGalleryCategoryChip<ImageView>])
+    case image(_ imageCategories: [BPKImageGalleryCategoryImage<ImageView>])
+}
+
+public struct BPKImageGalleryCategoryChip<ImageView: View> {
```

Comment:
Please move this struct inside the enum of the Style above. same for the Image struct below